sdk/src/loom2d/Display/DisplayObject.ls

```diff
@@ -322,6 +323,8 @@ package loom2d.display
             while (ancestor != this && ancestor != null)
                 ancestor = ancestor.parent;
 
+            _cachedStage = null;
```

Comment:
Yes, because otherwise we might retain a reference to the stage while not being on it (bad) and potentially have an invalid stage if there are more than one (not possible right now, but could break in the future). If we want to hardcode it to one stage, we could just have a static main stage var.